python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -1472,6 +1471,9 @@ def submit(self, keep_going=None, keep_on_fail=None, prepare=False):
             # split into subjobs
             rjobs = self._doSplitting()
 
+            if not rjobs or len(rjobs) == 1 and rjobs[0] is self and self.splitter is not None:
```

Comment:
I think it makes more sense to just check against `len(self.subjobs)` and if there is a splitter. I actually think this `rjobs` stuff probably isn't needed (or at least can be greatly simplified) but for that's probably for another PR.